examples/stencil/README.md

```diff
@@ -0,0 +1,146 @@
+# [Stencil](https://stenciljs.com)
+
+For a new Stencil project follow the [getting started](https://stenciljs.com/docs/getting-started) guide.
+
+To add Tailwind to a Stencil project, start by installing `@fullhuman/postcss-purgecss`, `@stencil/postcss`, `autoprefixer`, `cssnano` and `tailwindcss`:
+
+```sh
+npm install @fullhuman/postcss-purgecss @stencil/postcss autoprefixer cssnano tailwindcss --save-dev
+```
+
+Next, set up the PostCSS plugin in stencil.config.ts
+
+```js
+import { Config } from '@stencil/core';
+import { postcss } from '@stencil/postcss';
+import autoprefixer from 'autoprefixer';
+
+const purgecss = require('@fullhuman/postcss-purgecss')({
+  content: ['./src/**/*.tsx', './src/index.html'],
+  defaultExtractor: content => content.match(/[A-Za-z0-9-_:/]+/g) || []
+});
+
+export const config: Config = {
+  globalStyle: 'src/global/app.css',
+  outputTargets: [
+    {
+      type: 'www',
+      serviceWorker: null,
+      baseUrl: 'http://localhost:5000'
+    }
+  ],
+  plugins: [
+    postcss({
+      plugins: [
+        require('tailwindcss')('./tailwind.config.js'),
+        autoprefixer(),
+        ...(process.env.NODE_ENV === 'production'
+          ? [purgecss, require('cssnano')]
+          : [])
+      ]
+    })
+  ]
+};
+```
```

Comment:
You may want to update the config snippets with the sample config [here](https://github.com/tailwindcss/setup-examples/pull/24/files#diff-f52c70db92be6ed93dedb9cb2a3fbcb3).

Reply:
I've updated the example